### PR TITLE
[DEM-1141] features directive

### PR DESCRIPTION
**Pull request by vietnogi**
ping @appirio-tech/demandside

### CODE REVIEW

File: src/views/submit-work-features.directive.jade [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    li
+      checkbox(label="Dates & Locations")
+
+      button.clean example
+
+modal(show="vm.showDefineAFeatureModal")
+  form
+    input(type="text" placeholder="Feature name...")
+
```
Comment (samsep) [inline src/views/submit-work-features.directive.jade]: Think we also need an input for 'Feature Description', and maybe a placeholder for drag and drop

File: src/views/submit-work-features.directive.jade [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    li
+      checkbox(label="Registration")
+
+      button.clean example
+
+    li
+      checkbox(label="Dates & Locations")
+
+      button.clean example
+
```
Comment (samsep) [inline src/views/submit-work-features.directive.jade]: Also need a general button for 'Apply Features' once the user adds everything

File: src/views/submit-work-features.directive.jade (hunk 1 above)
Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: that stuff will not be implemented yet

File: src/views/submit-work-features.directive.jade (hunk 2 above)
Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: isn't that the checkbox?

Comment (samsep) [inline src/views/submit-work-features.directive.jade]: No in the new wireframes there's a button on the top-right for applying the features once the user's done, we'd need that to update the work & move on to the next phase

File: src/views/submit-work-features.directive.jade (hunk 1 above)
Comment (samsep) [inline src/views/submit-work-features.directive.jade]: We should have the feature description input as part of the modal though, this better matches the wireframes and is likely necessary

File: src/views/submit-work-features.directive.jade (hunk 2 above)
Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: oh, we're not doing that

Comment (samsep) [inline src/views/submit-work-features.directive.jade]: We should clarify w. product how the user finalizes features & moves on to the next stage then, currently not seeing anything in the designs

File: src/views/submit-work-features.directive.jade [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  select
+    option App Category
+
+    option option 1
+
+    option option 2
+
+    option option 3
+
```
Comment (samsep) [inline src/views/submit-work-features.directive.jade]: Not doing the dropdown as a feature for now so we can remove this

File: src/views/submit-work-features.directive.jade (hunk 1 above)
Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: When the user adds the description, it dosnt show up anywhere.

Comment (samsep) [inline src/views/submit-work-features.directive.jade]: But the description would still be stored & is needed as part of any custom feature, right? Not sure why we wouldn't have an input for it.

Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: minimal features to get a complete working flow then take slowly add features

Comment (samsep) [inline src/views/submit-work-features.directive.jade]: The description is part of the minimal working flow and fairly easy to implement, all default features have a description attached to them as well. We'll eventually need to add it so why not reduce work for later?

Comment (vietnogi) [inline src/views/submit-work-features.directive.jade]: cheese, you're really fighting for this one.  Ill add it in just to make your day.

**Pull request merged**